### ISSUE TITLE
Check jquery update

### ISSUE THREAD

**Issue by cosimameyer** [rladies/starter-kit-bookdown#2]
See the email from Dependabot - check whether we should update to jquery ~> 3.5.0

**Comment by drmowinckels** [comment]
right, this repo has been unecessary since the guide was made. #3 hints to archive it. I agree.
